tests/cpp/test_multidevice_host_ir_integration.cpp

```diff
@@ -0,0 +1,67 @@
+// clang-format off
+/*
+* SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+* All rights reserved.
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+// clang-format on
+#include <fusion.h>
+#include <host_ir/container.h>
+#include <host_ir/executor.h>
+#include <ir/all_nodes.h>
+#include <ops/all_ops.h>
+#include <tests/cpp/multidevice.h>
+
+namespace nvfuser {
+
+namespace hir {
+
+TEST_F(MultiDeviceTest, LaunchKernel) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+  TensorView* tv0 = makeSymbolicTensor(2);
+  fusion.addInput(tv0);
+
+  TensorView* tv1 = set(tv0);
+  fusion.addOutput(tv1);
+
+  auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+  at::Tensor t0 = at::randn({32, 32}, options);
+  std::vector<c10::IValue> aten_inputs = {t0};
+  auto ke = std::make_unique<KernelExecutor>();
+  ke->compile(&fusion, aten_inputs);
+
+  auto hic = std::make_unique<HostIrContainer>();
+  FusionGuard::setCurFusion(hic.get());
+
+  hic->pushBackKernelExecutor(ke.get());
+
+  IrCloner ir_cloner(hic.get());
+  auto tv2 = ir_cloner.clone(tv0);
+  auto tv3 = ir_cloner.clone(tv1);
+
+  std::vector<Val*> lk_inputs = {tv2};
```

Comment:
```suggestion
  std::vector<Val*> launch_kernel_inputs = {tv2};
```
Or simply `input_tvs`

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules
> Minimize the use of abbreviations that would likely be unknown to someone outside your project (especially acronyms and initialisms)